packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { FC } from 'react';
 import {
+  TextInputCell,
```

Comment:
Added:

- columns: count this line/batch
- accessors: Hide the value if not counting, show if counting. I thought it would be nice to not blank out the values entered when toggling the count, but also wanted the line to look blank. Also didn't want to disable the line, as you can start entering in a line to trigger the toggling also.
- Setters: Set the count this line when editing (annoying because we don't want count this line to also be setting the value to true everytime we update the checkbox!)